packages/server/lib/screenshots.ts

```diff
@@ -350,6 +368,7 @@ const getPath = function (data, ext, screenshotsFolder, overwrite) {
     .chain(data.titles)
     .map(sanitizeToString)
     .join(RUNNABLE_SEPARATOR)
+    // @ts-ignore
```

Comment:
I'm not sure what the next line is even doing; `_.concat` only operates on arrays, and `"someString".concat([]) === "someString"`. The next line can likely be removed, but I'd make sure the values are what they're expected to be if you do.

Reply:
Ryan had some thoughts on what this is doing here: https://github.com/cypress-io/cypress/pull/30758#discussion_r1884175371

Reply:
@cacieprins @ryanthemanuel Yah....it does not like any changes to this area. I'm going to leave it alone. 😬 I don't really understand why it's needed either.

Reply:
Yeah I think I get what it's doing, but lodash types don't like it even though you *can* do it.

Reply:
Hey, while I was working on this part of code in my PR, I found that this is done so that type of the variable is same in both if and else branch. I guess that is why it breaks, as the type for `name` variable becomes different in case of if and else branches, and the remaining code that uses `names` breaks on one type or other. I fixed it by replacing the concat by putting it in array.

Relevant lines from my PR https://github.com/cypress-io/cypress/pull/30673/files#diff-471ce7b0d88e3cdc126b6c5e781d074d86a828e5a7d88b9584db3385be09e569R88-R97